htlcswitch/link.go

```diff
@@ -570,7 +570,8 @@ func (l *channelLink) WaitForShutdown() {
 func (l *channelLink) EligibleToForward() bool {
 	return l.channel.RemoteNextRevocation() != nil &&
 		l.ShortChanID() != hop.Source &&
-		l.isReestablished()
+		l.isReestablished() &&
+		!l.IsFlushing()
```

Comment:
👍 

This'll prevent the router from trying to send an HTLC over the link, and also the switch from trying to forward over the link.